sdk/tables/Azure.Data.Tables/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+- Removed client side validation which prevented `GetEntity` and `GetEntityAsync` from getting an entity with an empty string as its RowKey value. ([#32447](https://github.com/Azure/azure-sdk-for-net/issues/32447))
```

Comment:
The fix also enables empty partition key again which is not mentioned here

Reply:
I'll update this when I prepare the release next week.